Automattic/jetpack
# Release Management: add bin folder to svnignore

#### Changes proposed in this Pull Request:

* I don't believe we need any of the files in that directory shipped to site owners.


#### Testing instructions:

* Run `./tools/deploy-to-svn.sh x.x`, x.x being a new tag.
* The `bin directory should not be brought along.

#### Proposed changelog entry for your changes:

* None
